tests/fake_memcached.py

```diff
@@ -0,0 +1,34 @@
+import time
+from typing import Dict, Optional, Union
+
+from aiomcache.client import Client
+
+
+class FakeAsyncMemcached(Client):
```

Comment:
maybe create a utils file and put this in? or a mocks folder, if we need this.

Reply:
Mocks folder it is, since we already have a `utils` folder because we're following the same naming structure as the starlite module.